api/tests/Unit/Command/UserRetentionPolicyCommandTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace App\Tests\Unit\Command;
+
+use App\Command\UserRetentionPolicyCommand;
+use App\Entity\User;
+use App\Event\UserRetentionPolicyCommandEvent;
+use App\EventDispatcher\ObservableEventDispatcher;
+use App\Repository\UserRepository;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class UserRetentionPolicyCommandTest extends KernelTestCase
+{
+    use ProphecyTrait;
+
+    public function setUp(): void
+    {
+        $kernel = static::createKernel();
+        $app = new Application($kernel);
+
+        $this->userRepository = self::prophesize(UserRepository::class);
+        $this->eventDispatcher = self::prophesize(ObservableEventDispatcher::class);
+        $this->logger = self::prophesize(LoggerInterface::class);
+
+        $sut = new UserRetentionPolicyCommand(
+            $this->userRepository->reveal(),
+            $this->eventDispatcher->reveal(),
+            $this->logger->reveal()
+        );
+
+        $app->add($sut);
+
+        $command = $app->find(UserRetentionPolicyCommand::$defaultName);
+        $this->commandTester = new CommandTester($command);
+    }
+
+    /**
+     * @test
+     */
+    public function testOutputWithNoInactiveAdminUsers()
+    {
+        $this->userRepository->getAllAdminAccountsNotUsedWithin('-24 months')
+            ->shouldBeCalled()
+            ->willReturn(null);
+
+        $result = $this->commandTester->execute([]);
+        $this->assertEquals(0, $result);
+    }
+
+    /**
+     * @test
+     */
+    public function testExecuteWithInactiveAdminUser()
```

Comment:
I think all of your tests here, should at least include a standard user along side the admin accounts. I know you've got this partially covered in the next test, but we should have absolute confidence that no normal users are currently touched.